features/fields_extensie.feature

```diff
@@ -1,126 +1,544 @@
+      En bevat de response een invalidParams met de volgende gegevens
+      | code     | name          | reason                                                |
+      | required | fields        | Parameter bevat een porpertynaam met onjuiste casing. |
+
+    Scenario: opvragen veld met fields door opgeven naam veld die meerdere keren in de resource voorkomt
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999994086 |
+      | geslachtsaanduiding | M         |
+      En de persoon heeft de volgende geboorte gegevens
+      | naam                                      | waarde   |
+      | datum (01.03.10)                          | 19860401 |
+      | land.code                                 | 6030     |
+      | plaats.code                               | 0518     |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                   |
+      | type                | RaadpleegMetBurgerservicenummer          |
+      | burgerservicenummer | 999994086                                |
+      | fields              | code                                     |
+      Dan bevat de response de volgende gegevens
+      | naam     | waarde                                                                                                      |
+      | type     | https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?#System_Net_HttpStatusCode_BadRequest |
+      | title    | Een of meerdere parameters zijn niet correct.                                                               |
+      | status   | 400                                                                                                         |
+      | detail   | De foutieve parameter(s) zijn: fields.                                                                      |
+      | code     | paramsValidation                                                                                            |
+      | instance | /personen                                                                                                   |
+      En bevat de response een invalidParams[0] met de volgende gegevens
+      | code     | name          | reason              |
+      | required | fields        | Deel van de parameterwaarde niet correct: geboorte.land.code  |
+      En bevat de response een invalidParams[1] met de volgende gegevens
+      | code     | name          | reason              |
+      | required | fields        | Deel van de parameterwaarde niet correct: geboorte.plaats.code  |
+
+  Rule: De response bevat alleen de properties die in de fields parameter gevraagd en indien relevant de volgende properties :
+        De volgende velden worden geleverd ongeacht of de fields parameter is gebruikt en ongeacht de eventueel gevraagde velden in de fields parameter:
```

Comment:
Ongeacht of de field parameter is gebruikt is denk ik niet nodig want die is verplicht

Reply:
Aangepast.